lib/danger/request_sources/github/github.rb

```diff
@@ -343,8 +343,10 @@ def submit_inline_comments_for_kind!(kind, messages, diff_lines, danger_comments
 
           if matching_comments.empty?
             begin
+              # Since Octokit v8, the signature of create_pull_request_comment has been changed.
+              # See https://github.com/danger/danger/issues/1475 for detailed information.
               client.create_pull_request_comment(ci_source.repo_slug, ci_source.pull_request_id,
-                                                 body, head_ref, m.file, position)
+                                                 body, head_ref, m.file, (Octokit::MAJOR >= 8 ? m.line : position))
```

Comment:
`Octokit::MAJOR` exists since [Octokit v4.0.0](https://github.com/octokit/octokit.rb/blob/v4.0.0/lib/octokit/version.rb) and it is the minimum version that Danger supports. https://github.com/danger/danger/blob/bd89c4d6f3c7650d766eb804000a10f0b2fb98ab/danger.gemspec#L31